apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -125,164 +237,121 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     return res.status(403).json({ message: "Signature does not match" });
   }
 
-  const response = schema.safeParse(req.body);
-
   log.debug(
-    "Daily video recording webhook Request Body:",
+    "Daily video webhook Request Body:",
     safeStringify({
-      response,
+      body: req.body,
     })
   );
 
-  if (!response.success || response.data.type !== "recording.ready-to-download") {
-    return res.status(400).send({
-      message: "Invalid Payload",
-    });
-  }
+  try {
+    if (req.body?.type === "recording.ready-to-download") {
```

Comment:
this if else statement would handle 
1) "recording.ready-to-download" event
2) "meeting.ended" event